test.js

```diff
@@ -1,36 +1,59 @@
 'use strict';
-var assert = require('assert');
+var assert = require('chai').assert;
```

Comment:
I commented on the `npm` IRC channel, http://logs.nodejs.org/npm/2015-07-29#01:12:00.530, but haven't received a response yet. I'll follow up with that channel later to see what the community thinks of the issue.